packages/lwt-windows.2.5.2/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "1.2"
+maintainer: "Pierre Boutillier <pierre.boutillier@laposte.net>"
+authors: [
+  "Jérôme Vouillon"
+  "Jérémie Dimino"
+]
+homepage: "http://github.com/ocsigen/lwt"
+dev-repo: "git://github.com/ocsigen/lwt"
+bug-reports: "http://github.com/ocsigen/lwt/issues"
+license: "LGPL with OpenSSL linking exception"
+
+build: [
+  ["ocaml" "setup.ml" "-configure"
+    "--override" "os_type" "Win32"
+    "--override" "flexlink" "%{lib}%/flexdll-windows/flexlink"
+    "--override" "ext_dll" ".dll"
+    "--override" "datarootdir" "%{prefix}%/windows-sysroot/share"
+    "--disable-pthread"
```

Comment:
It is copy/paste from https://github.com/vouillon/opam-windows-repository/blob/master/packages/win64-lwt/win64-lwt.2.4.5/files/build.sh 
Naive compilation with --enable-pthread fails with
```
/home/pirbo/.opam/4.02.3/bin/ocamlfind ocamlc -g -ccopt -I/usr/include -ccopt -Isrc/unix -c src/unix/lwt_unix_stubs.c
+ /home/pirbo/.opam/4.02.3/bin/ocamlfind ocamlc -g -ccopt -I/usr/include -ccopt -Isrc/unix -c src/unix/lwt_unix_stubs.c
In file included from /usr/include/stdlib.h:24:0,
                 from /home/pirbo/.opam/4.02.3/windows-sysroot/lib/ocaml/caml/misc.h:27,
                 from /home/pirbo/.opam/4.02.3/windows-sysroot/lib/ocaml/caml/alloc.h:21,
                 from src/unix/lwt_unix_stubs.c:28:
/usr/include/features.h:385:23: fatal error: gnu/stubs.h: Aucun fichier ou dossier de ce type
compilation terminated.
Command exited with code 2.
```